extensions/amp-story-shopping/0.1/amp-story-shopping.js

```diff
@@ -3,6 +3,41 @@ import {AmpStoryShoppingConfig} from './amp-story-shopping-config';
 import {AmpStoryShoppingTag} from './amp-story-shopping-tag';
 
 import {CSS as shoppingCSS} from '../../../build/amp-story-shopping-0.1.css';
+import {dependsOnStoryServices} from '../../amp-story/1.0/utils';
+
+/**
+ * Language and currency sensitive number formatting.
+ * @param {!../../../src/service/localization.LocalizationService} localizationService
+ * @param {!Element} el
+ * @param {string} currency An ISO 4217 currency code.
+ * @param {number} price
+ * @return {string}
+ */
+export const formatI18nNumber = (localizationService, el, currency, price) => {
+  const langCode = localizationService.getLanguageCodesForElement(el)[0];
+  try {
+    return new Intl.NumberFormat(langCode, {
+      style: 'currency',
+      currency,
+    }).format(price);
+  } catch {
+    return price.toString();
```

Comment:
Good fallback! I think we could also add the currency to it, just the raw data we got from the publisher, e.g. `${price} ${currency}`